README.md

```diff
@@ -275,6 +275,93 @@ Installation instructions assets-iac-terraform-azure by running:
     }
 ```
 
+### K8s Module
+
+```terraform
+    terraform {
+        required_providers {
+            azurerm = {
+                source = "hashicorp/azurerm"
+                version = ">= [version]"
+            }
+        }
+    }
+
+    provider "azurerm" {
+        features {}
+    }
+
+    resource "azurerm_resource_group" "rg" {
+        name = "k8sSample"
+        location = "West Europe"
+        tags =  {
+            MyTag = "MyTag value"
+        }
+    }    
+
+    module "k8s" {
+        source = "../../modules/k8s"
+        resource_group_name = azurerm_resource_group.rg.name
+        location = azurerm_resource_group.rg.location
+        k8s_name = "k8sSample"
+        dns_prefix = "k8sSample"
+        default_node_pool_name = "default"
+        default_node_pool_node_count = 3
+        default_node_pool_vm_size = "standard_d2_v2"
+        default_node_pool_disk_size_gb = 30
+
+        tags =  {
+            MyTag = "MyTag value"
+        }
+    }
+
+```
+
+### K8s Module with Container registry
```

Comment:
I have doubts here. A container registry aims to have "everything": a full general catalog of images. This leads me to questioning if having 1AKS instance with a single ACR every time might lead to extra and unneeded costs. On the other hand, I see the benefit of deploying AKS and ACR together when rolling out a cluster that might contain several projects at the same time. Thoughts? 😄